client/state/themes/selectors.js

```diff
@@ -45,6 +45,15 @@ export const getTheme = createSelector(
 		if ( siteId === 'wpcom' || siteId === 'wporg' ) {
 			return theme;
 		}
+
+		if ( ! theme ) {
```

Comment:
Just noting here that this comment was on a previous version of the PR where we had some interesting logic here.